### PR TITLE
OCM-10279 | test: automate id:75534

### PR DESCRIPTION
**Pull request by AkashKanni**
https://privatebin.corp.redhat.com/?c3213f53676be6f7#ErQ7P2pip2EDwQnxPbMtozhA6JGr5XdZwbzhV51rkJnT

### CODE REVIEW

File: tests/e2e/hcp_cluster_test.go [hunk 1]
```diff
@@ -340,4 +365,57 @@ var _ = Describe("HCP cluster testing",
...
+
+				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ")
+				rosalCommand := config.GenerateCommand(command)
+
+				stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+				Expect(err).To(BeNil())
+				Expect(stdout.String()).To(ContainSubstring(fmt.Sprintf("Cluster '%s' has been created", clusterName)))
+
+				By("wait for 5 minutes to install logs show up")
+				time.Sleep(5 * time.Minute)
```
Comment (radtriste) [inline tests/e2e/hcp_cluster_test.go:385]: Wouldn't it be better to wait for the `installing` state of the cluster ?

File: tests/e2e/hcp_cluster_test.go [hunk 2]
```diff
@@ -340,4 +365,57 @@ var _ = Describe("HCP cluster testing",
...
+				Expect(output.String()).Should(ContainSubstring("hostedclusters %s Release image is valid", clusterName))
+				Expect(output.String()).Should(ContainSubstring("Cluster '%s' is now ready", clusterName))
+
+				By("Delete the Hypershift cluster")
+				output, err = clusterService.DeleteCluster(clusterName, "-y")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("Cluster '%s' will start uninstalling now", clusterName))
+
+				By("wait for 2 minutes to uninstall logs show up")
+				time.Sleep(2 * time.Minute)
```
Comment (radtriste) [inline tests/e2e/hcp_cluster_test.go:406]: Wouldn't it be better to wait for the `uninstalling` state of the cluster ?

File: tests/e2e/hcp_cluster_test.go (hunk 1 above)
Comment (AkashKanni) [reply tests/e2e/hcp_cluster_test.go:385]: i tried 
```
err = clusterService.WaitClusterStatus(clusterID, "installing", 5, 24)
Expect(err).To(BeNil(), "It met error or timeout when waiting cluster to installing status")
```
but constantly facing error
`It met error or timeout when waiting cluster to installing status`

but at the same time, if we track the cluster status, the cluster gets installed successfully without any issues

Comment (radtriste) [reply tests/e2e/hcp_cluster_test.go:385]: @AkashKanni Let's keep it that way for now.
Could you create a follow-up card to review this later ?

Comment (AkashKanni) [reply tests/e2e/hcp_cluster_test.go:385]: Hey @radtriste i have created a ticket here https://issues.redhat.com/browse/OCM-11613, PTAL Thanks!

**Pull request merged**